### PR TITLE
Improve container support

### PR DESCRIPTION
**Pull request by onurmercury**
- Switch to Python container image
- Switch to Pyrofork
- Introduce Compose file support, container registry support and Actions support for possible future use

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
+asyncio
 langchain_community
-https://github.com/KurimuzonAkuma/pyrogram/archive/dev.zip
 psutil
-asyncio
-TgCrypto
\ No newline at end of file
+pyrofork
```
Comment (ket0x4) [inline requirements.txt:4]: why ?

**Review by ket0x4 — CHANGES_REQUESTED** [review]
ollamayı container'a dahil mi etsek ?

File: requirements.txt (hunk 1 above)
Comment (onurmercury) [reply requirements.txt:4]: available on pypi, prebuilt dependency, doesn't require any extra compilation.

installation times:
pyrofork
```real    0m 16.42s
user    0m 9.88s
sys     0m 4.92s```

current one
```real    0m 28.01s
user    0m 15.30s
sys     0m 9.53s```

**Comment by onurmercury** [comment]
> ollamayı container'a dahil mi etsek ?

halihazırda container olarak var, ayrı servis olarak compose a eklenebilir

**Pull request merged**